Author: Chris Kuehl <ckuehl@ocf.berkeley.edu>
Subject: Rename validator scripts
Forwarded: https://github.com/pre-commit/pre-commit/pull/251
Applied-Upstream: > 0.5.4, https://github.com/pre-commit/pre-commit/commit/b07572e32
Description: Rename validator scripts to be less generic and more suitable for
 inclusion into Debian.

diff --git a/setup.py b/setup.py
index c75a891..3c90049 100644
--- a/setup.py
+++ b/setup.py
@@ -51,8 +51,8 @@ setup(
     entry_points={
         'console_scripts': [
             'pre-commit = pre_commit.main:main',
-            'validate-config = pre_commit.clientlib.validate_config:run',
-            'validate-manifest = pre_commit.clientlib.validate_manifest:run',
+            'pre-commit-validate-config = pre_commit.clientlib.validate_config:run',
+            'pre-commit-validate-manifest = pre_commit.clientlib.validate_manifest:run',
         ],
     },
 )
